pkg/check/manifest/manifest.go

```diff
@@ -91,7 +91,7 @@ func (c *Check) Run(ctx context.Context, cluster orchestration.Cluster, opts int
 		return err
 	}
 
-	sortedNodes := cluster.NodeNames()
+	sortedNodes := cluster.FullNodeNames()
```

Comment:
Are the nodes really sorted? Do they need to be?

Reply:
I'm not sure. But the order is irrelevant